BDArmory/Control/MissileFire.cs

```diff
@@ -4506,7 +4532,7 @@ bool SmartPickWeapon_EngagementEnvelope(TargetInfo target)
                         {
                             MissileLauncher Bomb = item.Current as MissileLauncher;
                             if (targetWeapon != null && targetWeapon.GetWeaponClass() == WeaponClasses.Missile) continue;
-                            if (firedMissiles >= maxMissilesOnTarget) continue;// Max missiles are fired, try another weapon
+                            //if (firedMissiles >= maxMissilesOnTarget) continue;// Max missiles are fired, try another weapon
```

Comment:
commenting these out so if a craft only has missiles, the smartFindTarget can still find a target instead of failing the SmartPickWeapon check; attempting to launch when firedMissiles > =maxMissilesontarget is checked elsewhere.